docs/matrix-hal/examples/everloop.md

```diff
@@ -0,0 +1,499 @@
+<h2 style="padding-top:0">Everloop</h2>
+<h4 style="padding-top:0">HAL Example</h4>
+
+### Device Compatibility
+
+<img class="creator-compatibility-icon" src="../../img/creator-icon.svg">
+<img class="voice-compatibility-icon" src="../../img/voice-icon.svg">
+
+## Overview
+
+The Everloop interface supports:
+
+- Setting the RGBW colors for each individual LED.
+
+## Code Examples
+
+Function references can be found [here](/matrix-hal/reference/everloop).
+

```

Comment:
You should remove the compile command for each example for each interface and add it right below the Code Examples initial text

The command below will compile each example. Be sure to pass in your C++ file and desired output file in the command.
```language-bash
g++ -o YOUR_OUTPUT_FILE YOUR_CPP_FILE -std=c++11 -lmatrix_creator_hal
```

Reply:
Do this for each example page